docs/pom.xml

```diff
@@ -29,8 +29,17 @@
     <artifactId>keycloak-docs-parent</artifactId>
     <packaging>pom</packaging>
 
-    <modules>
-        <module>maven-plugin</module>
-        <module>guides</module>
-    </modules>
+    <profile>
```

Comment:
Missing tag `<profiles>`

Reply:
Are you saying that I have to provide a valid pom file? ;)

Reply:
Exactly :grin:

Reply:
Should be fixed now, can you check and approve if it's okay?

Reply:
Great progress! Well done!